translations/fr/pack/fhv/fhvp.json

```diff
@@ -0,0 +1,992 @@
+    "traits": "Allié. Science."
+  },
+  {
+    "code": "10042",
+    "name": "Microscope",
+    "slot": "Main",
+    "text": "Après qu'un ennemi dans votre lieu a été vaincu ou qu'un investigateur lui a échappé avec succès, inclinez le Microscope : placez 1 ressource sur le Microscope, en tant que preuve.\n[action] [action]: <b>Enquêter.</b> Vous gagnez +1 [intellect] pour cette enquête pour chaque preuve sur le Microscope (+3 [intellect] max.). En cas de réussite, vous pouvez dépenser jusqu'à 2 preuves pour découvrir un même nombre d'indices supplémentaires dans votre lieu.",
+    "traits": "Objet. Outil. Science."
+  },
+  {
+    "code": "10043",
+    "name": "Masque de la Souris",
+    "subname": "La Douce Observatrice",
+    "text": "Limite de 1 [[Masque]] par investigateur.\nUtilisations (2 offrandes). Reconstituez 1 de ces offrandes après que vous avez révélé un lieu ou que vous avez mis en jeu un nouveau lieu.\n[fast] Dépensez 1 offrande: Vous gagnez +2 [willpower] ou +2 [intellect] pour ce test de compétence. (Limite d'une fois par test.)",
+    "traits": "Objet. Charme. Masque."
+  },
+  {
+    "code": "10044",
+    "name": "Myconide Affamé",
+    "subname": "Non-identifié",
+    "text": "Limite 1 par deck.\n[action]: cherchez Croissance Surnaturelle parmi vos cartes liées et ajoutez-la à votre main.\n[action] Si le Myconide Affamé a au moins 3 croissances sur lui : déplacez chaque croissance vers votre réserve de ressources, en tant que ressources. Dans votre Carnet de Campagne, inscrivez que « vous avez découvert une nouvelle espèce. »",
+    "traits": "Créature. Monstre. Flore. Science."
+  },
+  {
+    "bonded_to": "Myconide Affamé",
+    "code": "10045",
+    "name": "Croissance Surnaturelle",
+    "text": "Liée (Myconide Affamé).\n<b>Enquêter.</b> Après la résolution de ce test, placez 1 ressource sur le Myconide Affamé, en tant que croissance, pour chaque point de réussite. Mettez de côté Croissance Surnaturelle, hors jeu. En cas d'échec, renvoyez Croissance Surnaturelle dans votre main.",
+    "traits": "Perspicacité. Science."
+  },
+  {
+    "code": "10046",
+    "flavor": "« Il injectait une certaine solution dans les veines de ses cadavres qui, s'ils étaient suffisamment frais, réagissaient d'étranges manières. »\n-H.P. Lovecraft, <i>Herbert West Réanimateur</i>",
+    "name": "Variable de Contrôle",
+    "text": "Rapide. À jouer après qu'un investigateur a révélé un pion [bless] lors d'un test de compétence dans votre lieu.\nDécouvrez 1 indice dans votre lieu.",
+    "traits": "Perspicacité. Science. Maudit."
+  },
+  {
+    "code": "10047",
+    "name": "Batterie de Tests",
+    "text": "En tant que coût supplémentaire pour jouer Batterie de Tests, dépensez une action.\n<b>Enquêter.</b> L'un après l'autre, enquêtez une fois dans votre lieu et dans un lieu connexe dans l'ordre croissant de leur valeur occulte, jusqu'à obtenir soit un échec soit un succès d'enquête dans chaque lieu éligible.",
```

Comment:
"et dans un lieu connexe" should be "et dans chaque lieu connexe", cf https://arkhamdb.com/card/10047 and https://cdn.discordapp.com/attachments/940926561618649118/1228460176407920691/image.png?ex=662c1fb1&is=6619aab1&hm=c48581c1aa805841b88472687d47d44346e7d6603fc05e6a48cdd4e471f87e5b& (not sure the latter link will be accessible)